index.html

```diff
@@ -0,0 +1,1008 @@
+              <li class="article-link">
+                <a href="#">Report: iOS 14 code suggests Apple is working on an iPhone 9 Plus too</a>
+              </li>
+              <li class="article-link">
+                <a href="#">Report: iOS 14 code suggests Apple is working on an iPhone 9 Plus too</a>
+              </li>
+            </ul>
+          </div>
+        </article>
+        <article class="category-article">
+          <h2 class="section-header">
+            <a href="#">
+              Apps
+              <span class="caret"></span>
+            </a>
+          </h2>
+          <div class="category-image">
+            <div class="inner-details">
+              <a href="#" class="picture-anchor"></a>
+              <div class="information">
+                <h5>MOST POPULAR</h5>
+                <a href="#">Report: iOS 14 code suggests Apple is working on an iPhone 9 Plus too</a>
+                <span>
+                  <a href="#">Napier Lopez</a>
+                  <span>&middot;</span>
+                  <span>21 hours ago</span>
+                </span>
+              </div>
+            </div>
+          </div>
+          <article class="latest-news-article category-latest-news">
+            <a href="#" class="article-image-anchor"></a>
+            <div class="article-details category-article-details">
+              <h4>
+                <a href="#">
+                  How to recover from burnout and chronic work stress, according to a psychologist
+                </a>
+              </h4>
+              <ul>
+                <li><a href="#">The Conversation</a></li>
+                <li><span>&middot;</span></li>
+                <li><span>10 minutes ago</span></li>
+              </ul>
+            </div>
+          </article>
+          <div class="article-links-holder">
+            <ul class="article-links">
+              <li class="article-link">
+                <a href="#">Report: iOS 14 code suggests Apple is working on an iPhone 9 Plus too</a>
+              </li>
+              <li class="article-link">
+                <a href="#">Report: iOS 14 code suggests Apple is working on an iPhone 9 Plus too</a>
+              </li>
+              <li class="article-link">
+                <a href="#">Report: iOS 14 code suggests Apple is working on an iPhone 9 Plus too</a>
+              </li>
+              <li class="article-link">
+                <a href="#">Report: iOS 14 code suggests Apple is working on an iPhone 9 Plus too</a>
+              </li>
+              <li class="article-link">
+                <a href="#">Report: iOS 14 code suggests Apple is working on an iPhone 9 Plus too</a>
+              </li>
+            </ul>
+          </div>
+        </article>
+      </section>
+
+      <section class="latest-deals">
+        <h2 class="section-article-heading">None</h2>
+
+        <header>
+          <h2>Latest Deals</h2>
+          <a href="#">See all Deals</a>
+        </header>
```

Comment:
- [ ] The titles don't have the right color. They should be red and bold.

Reply:
Thanks, done